### DISCUSSION TITLE
❓ Project questions / Preguntas del proyecto ❓

### DISCUSSION THREAD

**Discussion by Omitg24** [Arquisoft/lomap_es3a#41] [Ideas]
<p align="center">
<img src="https://user-images.githubusercontent.com/91057639/218811134-2eec75dc-a6d5-42f7-b1d6-fae6bd5ac158.png" width="640" height="320">
</p>

# EN: Discussion objective 
<p align="justify">
Here we will answer doubts or questions about aspects of the project, we will answer to anyone interested in it, from doubts of the developer team or the stakeholders.
</p>

# ES: Objetivo de la discusión
<p align="justify">
Aquí responderemos dudas o preguntas sobre aspectos del proyecto, responderemos a cualquier interesado en el mismo, desde dudas del equipo desarrollador o de los stakeholders.
</p>

**Comment by raulEspaa** [comment]
Buenas equipo! Estaba echándole un vistazo a los requisitos del proyecto y me surgió una duda, ya que no lo especifica entiendo que queda a nuestro criterio.

Los usuarios pueden tener amigos, grupos y demás. ¿Vosotros creéis que deberíamos añadir un usuario "Invitado" o "Anónimo"
aunque este no pueda interactuar o su información no se almacene para que pueda utilizar el mapa sin registrarse?

Un saludo,
Raúl

**Reply by Omitg24** [reply]
Puede ser una buena idea, es algo que realmente, se debería incluir siempre, en caso de que se quiera cumpllir con más requisitos, tales como la usabilidad o accesibilidad.
Por mi parte sería algo a valorar de incluir, sobre todo si vamos con tiempo suficiente y se puede añadir sin problemas, por lo que por mí estaría bastante bien si llegásemos a poder realizarlo.

Un saludo, 
Omar

**Reply by uo284373** [reply]
Buenas Raúl!
En mi opinión, estoy de acuerdo con Omar, en el sentido de que es una opción que cualquier sistema de este tipo debe incorporar. Si hay tiempo para hacerlo, sería una opción que incluiría sin ninguna duda.

Un saludo.
Carlos

**Reply by Nihilhatw** [reply]
En mi opinión creo que seria complicado llevarlo acabo, pues cada usuario "Anónimo" debería tener una cuenta independiente y a la hora de volver a acceder debería poder volver a consultar sus datos guardados.
Por ello me parece más simple que los usuarios tengan que registrarse con sus cuentas para poder usar las funcionalidades de la aplicación como incluir lugares o añadir amigos.

Un saludo,
David

**Comment by raulEspaa** [comment]
Buenas equipo! Tenéis idea de si el sistema debería de ser adaptable a dispositivos móviles? 
Lo digo por no liarla con los CSS

Un saludo,
Raúl

**Reply by Omitg24** [reply]
Eso creo que nos corresponde a nosotros decidir, podríamos añadirlo al final y ver si podemos adaptarlo, de manera extra, por lo que puede estar bien mirar.

Un saludo,
Omar

**Reply by Omitg24** [reply]
Finalmente se ha hecho adaptable, como comentaste @UO278036, tras estar trabajando en los CSS, en principio debería de ser más o menos (aun quedan un par de cosas por comprobar) usable en móviles!
